src/syft/ast/klass.py

```diff
@@ -294,6 +294,16 @@ def __init__(
         return_type_name: Optional[str],
         client: Optional[Any],
     ):
+        """
```

Comment:
Hi @dnabanita7!

This is not reflecting what the current implementation of Class is doing. If you want, we can setup a call to explain you the overall idea. Could you leave the docstring for Class to a separate PR? :)

Reply:
Actually, I have revert those chances in klass.py now. But yes I am all in for a call to understand the working.